django/db/models/base.py

```diff
@@ -1814,6 +1814,27 @@ def _check_long_column_names(cls):
     @classmethod
     def _check_constraints(cls):
         errors = []
+        for constraint in cls._meta.constraints:
```

Comment:
this part would be a function for constraints and indexes?

Reply:
Yes we can add `_check_object_identifiers()` and share the same errors:
```
* **models.E033**: The index/constraint name ``<index>`` cannot start with an underscore or a number.
* **models.E034**: The index/constraint name ``<index>`` cannot be longer than ``<max_length>`` characters.
```